apps/src/applab/applab.js

```diff
@@ -1932,6 +1932,7 @@ Applab.changeScreen = function(screenId) {
 };
 
 Applab.loadDefaultScreen = function() {
-  var defaultScreen = $('#divApplab .screen').first().attr('id');
+  var defaultScreen = $('#divApplab .screen[is-default=true]').first().attr('id')
+    || $('#divApplab .screen').first().attr('id');
```

Comment:
nit: Operator should go on previous line (see: https://google.github.io/styleguide/javascriptguide.xml#Code_formatting section Binary and Ternary Operators)